config/toggles.js

```diff
@@ -136,6 +136,10 @@ const longTermToggles = {
   // Disable HTTP, i.e. only use https
   requireHttps: environmentVariableTrue(pe, 'REQUIRE_HTTPS'),
 
+  // Toggle to redirect to different instance of refocus
+  enableRedirectDifferentInstance: environmentVariableTrue(pe,
+    'ENABLE_REDIRECT_DIFFERENT_INSTANCE'),
```

Comment:
@iamigo I like this toggle because it wraps around all of the functionality.

I do agree there are a lot of env vars here so I could take out SHOULD_REDIRECT_FOR_ROOMS and SHOULD_REDIRECT_FOR_PERSPECTIVES, and just assume that if REFOCUS_PERSPECTIVES_BASE_URL.length or REFOCUS_ROOMS_BASE_URL.length, then we want to redirect.